build.sbt

```diff
@@ -276,6 +271,35 @@ lazy val V = new {
     ).toList
 }
 
+// -Xlint is unusable because of
+// https://github.com/scala/bug/issues/10448
+def lintingOptions(scalaVersion: String) = {
+  val unused213 = "-Wunused"
+  val unused3 = "-Wunused:all"
+  val common = List(
+    // desugaring of for yield caused pattern var to complain
+    // https://github.com/scala/bug/issues/10287
+    "-Wconf:msg=parameter value .+ in anonymous function:silent",
+    // silence unused parameters in mtags
+    "-Wconf:src=*.ScaladocParser.scala&msg=in method reportError:silent",
```

Comment:
This seems pretty broad, or is it just me? Like are there more messages that the target one that include `in method` that we might be silencing that we don't want to?

Reply:
Sorry, could you rephrase the second sentence?

Reply:
Ha, yea sorry. I mean this will ignore any message that has `in method` in it right? If so, are there potentially multiple warnings that might have `in method` in it that we are ignoring when we specifically only want to ignore one? Just trying to make sure this isn't too broad.

Reply:
IIRC every message that comes from `ScaladocParser.scala` file and has `in method reportError` so it's a bit moe narrow. But I think I can change it to the `parameter (name1|name2) in method reportError` and this should be very precise.